vercel/turborepo
# publishing-libraries docs: Changesets configuration [#9324]

### Description

- Fixes #9324 
    
    The minimum changed need to make the recommended publishing script work is to set Changesets' `commit` option[^1] to true.


- Also adds a note about how to publish public packages. Changesets' default configuration is for private packages.

### Testing Instructions

1. Follow https://turbo.build/repo/docs/guides/publishing-libraries.
2. Confirm that the `publish-packages` script does not work to publish packages.
3. Make this PR's changes.
4. Confirm that the script now works.

[^1]: https://github.com/changesets/changesets/blob/7323704dff6e76f488370db384579b86c95c866f/docs/config-file-options.md?plain=1#L19
